site/content/sandbox.md

```diff
@@ -18,11 +18,15 @@ All services are exposed via the same ports as used in the guide.
 * **Play fair!** The sandbox's computing resources are (quite) limited. The number of devices that can be registered per tenant is therefore limited to 100.
 * The sandbox will be running the latest Hono release or milestone (if available). However, we may also deploy a more recent nightly build without further notice.
 * The sandbox exposes its API endpoints on both a TLS secured as well as an unsecured port. The secure ports use a Let's Encrypt certificate so you should not need to configure a specific trust store on your client in order to interact with them. Please note that when using the unsecured ports, the information you exchange with the sandbox might be exposed to eavesdroppers. We therefore **strongly suggest** that you use the secure ports only, if possible!
-  When using the [Hono client]({{< relref "/admin-guide/hono-client-configuration.md" >}}) to access the sandbox' Telemetry and/or Event APIs, make sure to not set a trust store explicitly but instead set the *tlsEnabled* property to `true`. The example below can be used to consume telemetry messages from the sandbox:
+  When using the [Hono client]({{< relref "/admin-guide/hono-client-configuration.md" >}}) to access the sandbox' Telemetry and/or Event APIs, make sure to not set a trust store explicitly but instead set the *tlsEnabled* property to `true`. The command line client binary downloaded ([link to downloads](https://www.eclipse.org/hono/download/#binaries)) can be used to consume telemetry/event messages from the sandbox as below:
 
     ```
-    $> java -jar target/hono-example-0.6-exec.jar --hono.client.host=hono.eclipse.org --hono.client.port=15671 --hono.client.tlsEnabled=true --hono.client.username=consumer@HONO --hono.client.password=verysecret --spring.profiles.active=receiver
+    $> java -jar hono-cli-*-exec.jar --hono.client.host=hono.eclipse.org --hono.client.port=15671 --hono.client.tlsEnabled=true --hono.client.username=consumer@HONO --hono.client.password=verysecret --spring.profiles.active=receiver
     ```
+{{% note %}}
```

Comment:
Have you checked how this is rendered in the web page?

Reply:
yes, as in the attached image.

![image](https://user-images.githubusercontent.com/18659982/48841763-ac81dc80-ed92-11e8-97c1-f5ebfc21fef0.png)

Reply:
It is hard to tell if the overall page's flow is still intact because the screenshot only shows a very small part of the page ...

Reply:
I hope this full page screen shot helps.
![sandbox](https://user-images.githubusercontent.com/18659982/48842647-0c798280-ed95-11e8-92f9-0e243654ff7b.png)

Reply:
yes, and it illustrates how the NOTE tag has destroyed the formatting of the remainder of the list :-(

Reply:
Thanks, I got it now.